library/kani/src/arbitrary.rs

```diff
@@ -170,3 +170,11 @@ where
         Box::new(T::any())
     }
 }
+
+impl Arbitrary for std::time::Duration {
+    fn any() -> Self {
+        const NANOS_PER_SEC: u32 = 1_000_000_000;
+        let nanos = u32::any() % NANOS_PER_SEC;
```

Comment:
How does this perform compared to using an assumption (`nanos < NANOS_PER_SEC`)?

Reply:
great question... I didn't spend time optimizing this implementation since my use case was fairly simple.

Reply:
I see. I'm worried that a modulo operation with a number that is not 2^n might be expensive.

Reply:
I added a performance test. The original version took around 14s and the assume one took close to 12s. So not a huge difference, but assume seems better. I also tried using a conditional assignment instead of a global assumption, but there wasn't any difference, so I'm keeping it simple for now and using the assume version.